uvm/servlets/admin/config/administration/MainController.js

```diff
@@ -88,7 +88,7 @@ Ext.define('Ung.config.administration.MainController', {
         Ext.Deferred.sequence([
             Rpc.asyncPromise('rpc.adminManager.getSettings'),
             Rpc.asyncPromise('rpc.systemManager.getSettings'),
-            Rpc.asyncPromise('rpc.skinManager.getSkinsList'),
+            //Rpc.asyncPromise('rpc.skinManager.getSkinsList'),
```

Comment:
Let's just delete it, not comment it out.

Reply:
removed